src/content/zh-tw/_project.yaml

```diff
@@ -0,0 +1,22 @@
+---
+announcement:
+  description: </#cdata-section></style><style><#cdata-section> 錯過了今年的 <b>Chrome 开发峰会</b>?
+    您可以在我們的YouTube播放列表觀看全部的峰会視頻。 <a href="\"https://www.youtube.com/playlist?list=PLNYkxOF6rcICUD5nBfRdAR6Fveosnqa5m\""
+    class="\"gc-analytics-event\"" data-category="\"webFu\"" data-label="\"cds-watch-playlist\"">观看</a>
+buganizer_id: '180451'
+color: google-blue
+content_license: cc3-apache2
+footer_path: "/web/_footer.yaml"
+google_analytics_ids:
+- UA-52746336-1
+home_url: "/web/"
+icon:
+  path: "/web/images/web-fundamentals-icon192x192.png"
+is_family_root: t
+name: Web
+tags:
+- html5
+- "\t\nmobile"
```

Comment:
Odd `\t\n` prepended
@sotayamashita

Reply:
@agektmr the reason is that some user add the HTML special characters.

Reply:
@sotayamashita do you mean in translation or in original document? If this happens in GitLocalize then you should probably trim them automatically in GitLocalize?

Reply:
In this case, it seems that a tab and a line feed were added by a user from the GitLocalize editor. In YAML file, we should consider that a string might have those characters intentionally, so if removing them it would raise another issue, I think.

Reply:
@chikathreesix Got it. Let's leave this for now as this PR is deprecated. I'll ping you next time this comes up as an issue.